src/style/themes/base/base_theme.config.js

```diff
@@ -0,0 +1,22 @@
+export default (palette) => {
+  const { atOpacity } = palette,
+      blackWithOpacity = atOpacity('#000000');
+
+  return (
+    {
+      colors: {
+        white: '#FFFFFF',
+        error: palette.errorRed,
+        warning: palette.gold,
+        sageLogo: palette.brilliantGreen,
+        success: palette.brilliantGreenShade(20),
+        info: palette.productBlueShade(3),
+        text: {
+          body: blackWithOpacity(0.9),
+          disabled: blackWithOpacity(0.55),
+          placeholder: blackWithOpacity(0.3)
+        }
+      }
+    }
+  );
+};
```

Comment:
can you keep file names consistent with dashes rather than the odd underscore please?